go.mod

```diff
@@ -104,3 +185,6 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+// remove when the https://github.com/kiegroup/container-builder/pull/18 will be merged
```

Comment:
all those dependency changes were introduced only by the builder version change?

Reply:
fixed